gcovr/tests/test_gcovr.py

```diff
@@ -233,6 +233,11 @@ def pytest_generate_tests(metafunc):
                 continue
 
             marks = [
+                pytest.mark.xfail(
+                    name == "simple1-drive-subst"
+                    and not is_windows,
+                    reason="drive substitution only available on windows",
+                ),
```

Comment:
I think this should be marked as `skip` rather than `xfail`. Xfail tests are still attempted, but a failure does not fail the entire test suite.

Reply:
You're right.

Reply:
I also change the others which will never work to `skip`.